docs/site/gpu.md

```diff
@@ -65,6 +81,79 @@ CUDA toolkit
   3. [CUDA 10.2](https://developer.nvidia.com/cuda-10.2-download-archive)
   4. [CUDNN 7.x](https://developer.nvidia.com/cudnn)
 
+## Linux
+
+## Linux
+
+One easiest way to install the NVIDIA software is with `apt` on Ubuntu. For other distributions
+refer to the [CUDA install Linux](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html).
+
+Note: All linux distributions may not support this. you might encounter some problems with driver
+installations.
+
+To check the CUDA compatible driver version:
+
+```sh
+
+# linux-modules-nvidia-390-5.0.0-1018-azure
+#                                    -aws, -gcp, -oracle, -generic, or -lowlatency
+
+# Get the latest driver version number, significant digit 5 like 435, 450
+NVIDIA_DRIVER_VERSION=$(sudo apt-cache search 'linux-modules-nvidia-[0-9]+-generic$' | awk '{print $1}' | sort | tail -n 1 | head -n 1 | awk -F"-" '{print $4}')
+
+CUDA_DRIVER_VERSION=$(apt-cache madison nvidia-cuda-toolkit | awk '{print $3}' | sort -r | while read line; do
+   if dpkg --compare-versions $(dpkg-query -f='${Version}\n' -W nvidia-driver-${NVIDIA_DRIVER_VERSION}) ge $line ; then
+       echo "$line"
+       break
+   fi
+done)
```

Comment:
These instructions do work. But, can this used in conjunction with cudnn install commands?